src/content/self-hosted/administration/configuration.mdx

```diff
@@ -150,6 +150,10 @@ The `allowDomains` field is optional. Only users with Google Workspace accounts
 
 Follow your OpenID Connect service provider's documentation on how to create the required application.
 
+We have custom guides for:
```

Comment:
This reads weird, since there is no context between this paragraph and the next.  Do we expect them to read those guides and then continue?  I'm not sure we should put this there.

Reply:
Maybe we can have two sections one of guides and a generic one?

Reply:
I would have them for discoverability, but yes, we should work on the wording. My thinking was, if you are on okta or azure AD, jump to their custom guides. For any other open id provider, continue reading here